samples/Snippets/Core/Snippets.cs

```diff
@@ -0,0 +1,148 @@
+﻿using Polly;
+using Polly.Retry;
+
+namespace Snippets.Core;
+
+#pragma warning disable IDE0022 // Use expression body for method
+#pragma warning disable IDE0021 // Use expression body for constructor
```

Comment:
Maybe just turn these off at the project level?

Reply:
Fixed in #1544

Reply:
Done.